src/daemon/notifications/dbus.vala

```diff
@@ -304,27 +303,29 @@
 		private void calculate_position(Popup window, Gdk.Rectangle rect, out int x, out int y) {
 			var pos = (NotificationPosition) this.panel_settings.get_enum("notification-position");
 			var latest = this.popups.get(this.latest_popup_id);
+			bool latest_exists = false;
+			int existing_height = 0, existing_x = 0, existing_y = 0;
+
+			if (latest != null && !latest.destroying) {
```

Comment:
Could do instead:

```
bool latest_exists = (latest != null && !latest.destroying);
int existing_height = 0, existing_x = 0, existing_y = 0;

if (latest_exists) {
[...]
}
```

Reply:
Good idea. I'll change that sometime this afternoon 😃